guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -33,7 +39,7 @@ For more information about backups, see https://access.redhat.com/documentation/
 ----
```

Comment:
While you're at it, could you hide the above section behind a katello/satellite definition? Vanilla Foreman doesn't need `rh-postgresql12-postgresql-evr`.